de.sesu8642.feudaltactics.yaml

```
@@ -0,0 +1,40 @@
+app-id: de.sesu8642.feudaltactics
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk17]
+command: feudaltactics
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

it required for X11 performance.